docs/Reference/Type-Providers.md

```diff
@@ -91,6 +92,41 @@ See also the [TypeBox
 documentation](https://github.com/sinclairzx81/typebox#validation) on how to set
 up AJV to work with TypeBox.
 
+### Zod
+
+The following sets up a Zod Type Provider
```

Comment:
I'd mention this is not an official package.

Reply:
I agree. And would we be adding a section here for every community module? I'm not sure this documentation belongs here. The module itself should explain how to use it.

Reply:
I agree, but discoverability is an important thing too.
Would it be better if code sample was replaced with a link to zod provider repo?

Reply:
I see it is already listed under https://www.fastify.io/ecosystem/. Are you wanting to create another such list in this document?

Reply:
this part of documentation already has a list, which gives a false impression of what type providers exist.

not everyone going to go through a random dump of ecosystem doc

Reply:
Surprising fact (😉): I have not paid this doc any attention. The only reason I have given any review of it this time is because you specifically asked me for one.

I think it is a mistake to have two lists with the same information in the documentation. In my opinion, we'd either note here that folks should review the ecosystem list for modules named as described, or update the ecosystem doc to _remove_ any type providers in that list and add a section that states "see this other doc for the list".

Reply:
what do you think about adding "type provider" section to ecosystem doc and linking it from here?

Reply:
Yeah, that sounds good too.